project_finder_web_app/models.py

```diff
@@ -0,0 +1,48 @@
+from django.db import models
+from datetime import date
+
+
+class Skill(models.Model):
+    skill = models.CharField(max_length=50, unique=True)
+
+    def __str__(self):
+        return self.skill
+
+
+class Hackathon(models.Model):
+    name = models.CharField(max_length=50, unique=True)
+    # creator = models.ForeignKey(User, on_delete=models.CASCADE)
+    hackathon_desc = models.TextField(max_length=500)
+    link = models.URLField()
+    hackathon_date = models.DateField(default=date.today)
+
+    def __str__(self):
+        return self.name
+
+
+class HackathonTeam(models.Model):
+    name = models.CharField(max_length=50, unique=True)
+    # leader = models.ForeignKey(User, on_delete=models.CASCADE)
+    # current_members = models.ManyToManyField(User)
+    hackathon = models.OneToOneField(Hackathon, on_delete=models.CASCADE)
```

Comment:
Why OneToOne? Would'nt that restrict to only one team per hackathon?

Reply:
I have changed it to a ManyToManyField.

Reply:
It needs to be Foreign Key. Please read up on OneToOneField, ForeignKey and ManyToMayField ka implementation.

Reply:
Okay done with it